mappings/net/minecraft/item/Item.mapping

```diff
@@ -155,7 +155,7 @@ CLASS net/minecraft/class_5009 net/minecraft/item/Item
 	METHOD method_25534 canModify ()Z
 	METHOD method_25535 register ()V
 	METHOD method_25536 getStackForRender ()Lnet/minecraft/class_5013;
-	CLASS class_5010 ToolMaterial
+	CLASS class_5010 ToolMaterials
```

Comment:
`ToolMaterials` is plural and there is only one class that holds tool materials, so `ToolMaterial` is fine

Reply:
Go check upstream

Reply:
upstream has an interface + enum
this just has the enum

Reply:
`ToolMaterials` is fitting for upstream because you always refer to `ToolMaterial`, the interface
here there is no interface but you still refer to it as a tool material, not tool materials

Reply:
Hmmm fair enough, i will revert this